### PR TITLE
Add a testsuite

### LINKED ISSUE #12: create an e3 testsuite driver
it's time.

### PR DESCRIPTION
**Pull request by setton**
Fixes #12

### CODE REVIEW

File: testsuite/testsuite.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+"""
+
+import os
+
+import e3.testsuite
+
+from drivers.shell_script import ShellScriptDriver
+
+
+class AFLControlTestsuite(e3.testsuite.Testsuite):
```
Comment (glacambre) [inline testsuite/testsuite.py]: Should probably be AdareduceControlTestsuite? :)

**Review by glacambre — DISMISSED** [review]

File: testsuite/testsuite.py (hunk 1 above)
Comment (setton) [reply testsuite/testsuite.py]: Copypasta. Fixed!

**Review by glacambre — APPROVED** [review]
LGTM!

File: testsuite/testsuite.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        super().set_up()
+
+        # Set PYTHONPATH to find adareducer
+        root_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
+        os.environ["PYTHONPATH"] = "{}:{}".format(
+            root_dir,
+            os.environ.get("PYTHONPATH", ""),
+        )
+
+        os.environ["ADAREDUCER"] = f"python {root_dir}/ada_reduce.py"
```
Comment (vser1) [inline testsuite/testsuite.py:33]: Is this really the way we go? Why can't we just add the dir to PATH? I've got traumatized of such environment variable abuses :)

Comment (setton) [reply testsuite/testsuite.py:33]: My only argument in favour of this is that it'll be convenient when we introduce coverage testing, we'll be able to just change this one line to a more complex command.
 
I don't think it'll pose a danger but perhaps I haven't been traumatized enough :-)

I don't have a strong opinion, feel free to open a PR to change this if you want.

**Pull request merged**